components/RichTextPageContent/index.js

```diff
@@ -144,31 +146,40 @@ export function getRichTextRenderOptions(links, options = defaultOptions) {
         const { title, url, height, width, description } = assetBlockMap.get(
           node.data.target.sys.id,
         );
-
-        return (
-          <div className={RichTextPageContentStyles.page__imgContainer}>
-            <Image
-              src={url}
-              alt={description}
-              height={height}
-              width={width}
-              layout="responsive"
-            />
-          </div>
-        );
+        if (renderNativeImg) {
+          return (
+            <div className={RichTextPageContentStyles.page__imgContainer}>
+              <img src={url} alt={description} height={height} width={width} />
+            </div>
+          );
+        } else {
+          return (
+            <div className={RichTextPageContentStyles.page__imgContainer}>
+              <Image
+                src={url}
+                alt={description}
+                height={height}
+                width={width}
+                layout="responsive"
+              />
+            </div>
+          );
+        }
       },
     },
   };
 }
 
 export default function RichTextPageContent(props) {
-  const { richTextBodyField, isBlogPost } = props;
+  const { richTextBodyField, renderH2Links } = props;
 
   return (
     <div className={RichTextPageContentStyles.page__content}>
       {documentToReactComponents(
         richTextBodyField.json,
-        getRichTextRenderOptions(richTextBodyField.links, { isBlogPost }),
+        getRichTextRenderOptions(richTextBodyField.links, {
```

Comment:
From the RichTextPageContent render on /buildrss, renderH2Links is coming in as 'undefined', being passed as 'undefined' and therefore not being overridden with ...defaultOptions.

Is there a way around this?